src/current/molt/molt-overview.md

```diff
@@ -0,0 +1,58 @@
+
+The MOLT tools can be used when:
+
+- Converting a schema for compatibility with CockroachDB.
+- Loading test and production data into CockroachDB.
+- Validating queries on CockroachDB.
+- Performing a cutover to finalize the migration to CockroachDB.
+
+## MOLT tools
+
+<img src="{{ 'images/molt/molt_tools.svg' | relative_url }}" alt="MOLT tooling overview" style="max-width:100%" />
+
+|                        Tool                       |          Supported databases          |                   Usage                   |                                      Release status                                      |
+|---------------------------------------------------|---------------------------------------|-------------------------------------------|------------------------------------------------------------------------------------------|
+| [Schema Conversion Tool](#schema-conversion-tool) | PostgreSQL, MySQL, Oracle, SQL Server | Schema conversion                         | GA                                                                                       |
+| [Fetch](#fetch)                                   | PostgreSQL, MySQL, CockroachDB        | Initial data load; continuous replication | GA                                                                                       |
+| [Verify](#verify)                                 | PostgreSQL, MySQL, CockroachDB        | Data validation                           | [Preview]({% link {{site.current_cloud_version}}/cockroachdb-feature-availability.md %}) |
+| [Live Migration Service](#live-migration-service) | PostgreSQL, MySQL, CockroachDB        | Consistent cutover                        | [Preview]({% link {{site.current_cloud_version}}/cockroachdb-feature-availability.md %}) |
+
+### Schema Conversion Tool
+
+The [MOLT Schema Conversion Tool]({% link cockroachcloud/migrations-page.md %}) [converts a source database schema]({% link {{site.current_cloud_version}}/migration-overview.md %}#convert-the-schema) to a CockroachDB-compatible schema. The supported Schema Conversion Tool sources are PostgreSQL, MySQL, Oracle, and SQL Server.
+
+The tool will convert [data definition (DDL) syntax]({% link {{site.current_cloud_version}}/sql-statements.md %}#data-definition-statements) (while ignoring destructive statements such as `DROP`), identify [unimplemented features and syntax incompatibilities]({% link {{site.current_cloud_version}}/migration-overview.md %}#unimplemented-features-and-syntax-incompatibilities) in the schema, and suggest edits according to CockroachDB [best practices]({% link {{site.current_cloud_version}}/migration-overview.md %}#schema-design-best-practices).
+
+### Fetch
+
+[MOLT Fetch]({% link molt/molt-fetch.md %}) moves data from a source database into CockroachDB. Data can be moved via one-time bulk ingestion and continuously via streaming ingestion. The supported Fetch sources are PostgreSQL, MySQL, and CockroachDB.
```

Comment:
```suggestion
[MOLT Fetch]({% link molt/molt-fetch.md %}) moves data from a source database into CockroachDB. Data is moved via one-time bulk ingestion which can optionally be followed by continuously streaming ingestion. The supported Fetch sources are PostgreSQL, MySQL, and CockroachDB.
```

Reply:
Just to clarify, we will eventually have other modes (continuous only, etc.) but i think adams suggestion works for now